compiler-plugin/src/main/kotlin/it/unibo/collektive/frontend/checkers/NoAlignInsideLoop.kt

```diff
@@ -15,33 +19,167 @@ import org.jetbrains.kotlin.fir.expressions.FirWhileLoop
  */
 object NoAlignInsideLoop : FirFunctionCallChecker(MppCheckerKind.Common) {
 
-    private val safeOperators = listOf("alignedOn", "align", "dealign")
+    private val SAFE_OPERATORS = listOf("alignedOn", "align", "dealign")
```

Comment:
I prefer the FQN for these functions.
If you have a custom function called `align` (with another meaning and semantics) called inside an `Aggregate` function this plugin raises a warning in any case since the function name is the same, right?
Instead only when the `align` coming from `it.unibo.collektive.aggregate.api.Aggregate` is called the warning should be raised.

Reply:
You're right, I will fix them using fully qualified names.